CHANGELOG.md

```diff
@@ -11,6 +11,111 @@ git log --first-parent
+
+Supported systems and requirements:
++ For all platforms, CMake 3.13+
++ C++14 compliant compiler
++ Linux and macOS
+  + GCC: 5.5+
+  + clang: 3.9+
+  + Intel compiler: 2018+
+  + Apple LLVM: 8.0+
+  + NVHPC: 22.7+
+  + Cray Compiler: 14.0.1+
+  + CUDA module: CUDA 9.2+ or NVHPC 22.7+
+  + HIP module: ROCm 4.5+
+  + DPC++ module: Intel OneAPI 2021.3 with oneMKL and oneDPL. Set the CXX compiler to `dpcpp`.
++ Windows
+  + MinGW and Cygwin: GCC 5.5+
+  + Microsoft Visual Studio: VS 2019
+  + CUDA module: CUDA 9.2+, Microsoft Visual Studio
+  + OpenMP module: MinGW or Cygwin.
+
+### Version Support Changes
++ ROCm 4.0+ -> 4.5+ after [#1303](https://github.com/ginkgo-project/ginkgo/pull/1303)
+
+### Interface Changes
++ Due to internal changes, `ConcreteExecutor::run` will now always throw if the corresponding module for the `ConcreteExecutor` is not build [#1234](https://github.com/ginkgo-project/ginkgo/pull/1234)
++ Changes the constructor of `experimental::distributed::Vector` to only accept local vectors as `std::unique_ptr` [#1284](https://github.com/ginkgo-project/ginkgo/pull/1284)
++ Changes some default parameter for the `solver::MultiGrid`. In particular, the smoother default to on iteration of `Ir` with `Jacobi` preconditioner, and the coarse grid solver uses the new direct solver with LU factorization. [#1291](https://github.com/ginkgo-project/ginkgo/pull/1291) [#1327](https://github.com/ginkgo-project/ginkgo/pull/1327)
++ Adds a more expresive callback to the `iteration_complete` event and deprecates the old ones. [#1288](https://github.com/ginkgo-project/ginkgo/pull/1288) [#1327](https://github.com/ginkgo-project/ginkgo/pull/1327)
+
+### Deprecations
+
++ Deprecated less expressive `iteration_complete` event. Users are advised to now implement the function `void iteration_complete(const LinOp* solver, const LinOp* b, const LinOp* x, const size_type& it, const LinOp* r, const LinOp* tau, const LinOp* implicit_tau_sq, const array<stopping_status>* status, bool stopped)` [#1288](https://github.com/ginkgo-project/ginkgo/pull/1288)
+
+### Added Features
++ Add a distributed Schwarz preconditioner. [#1248](https://github.com/ginkgo-project/ginkgo/pull/1248)
++ Flexible Gmres solver [#1244](https://github.com/ginkgo-project/ginkgo/pull/1244)
++ Enabled Gmres solver for distributed matrices and vectors [#1201](https://github.com/ginkgo-project/ginkgo/pull/1201)
++ Example that uses Kokkos to assemble the matrix [#1216](https://github.com/ginkgo-project/ginkgo/pull/1216)
++ Symbolic LU factorization allowing the `gko::experimental::factorization::Lu` and `gko::experimental::solver::Direct` classes to be used for matrices with non-symmetric sparsity pattern [#1210](https://github.com/ginkgo-project/ginkgo/pull/1210)
++ Numerical Cholesky factorization [#1215](https://github.com/ginkgo-project/ginkgo/pull/1215)
++ Add GCR solver [#1239](https://github.com/ginkgo-project/ginkgo/pull/1239)
```

Comment:
```suggestion
+ Added GCR solver [#1239](https://github.com/ginkgo-project/ginkgo/pull/1239)
```
because the others uses `added`. Or, change the others to `add`